### PR TITLE
role to update the run scripts to run on core 2

### PR DESCRIPTION
**Pull request by troy2914**
and cleanup for finals-dhaka

### CODE REVIEW

File: script/build_hosts [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
     max_team = 152
     max_human_judge = 12
-    max_pc2_autojudge = 0
-    max_kattis_autojudge = 16
+    max_pc2_autojudge = 16
+    max_kattis_autojudge = 0
     max_domjudge = 16
     max_domjudge_ccsadmin = 5
-    max_ccsadmin = 5
+    max_ccsadmin = 4
```
Comment (meisterT) [inline script/build_hosts:22]: What is this for?

File: script/build_hosts [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+if args.wf:
     name = "wf2021"
     timezone = "Asia/Dhaka"
     max_team = 152
     max_human_judge = 12
-    max_pc2_autojudge = 0
-    max_kattis_autojudge = 16
+    max_pc2_autojudge = 16
+    max_kattis_autojudge = 0
     max_domjudge = 16
```
Comment (meisterT) [inline script/build_hosts:20]: for consistency, should we use `max_domjudge_autojudge` here?

File: script/build_hosts [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
     max_domjudge_ccsadmin = 5
```
Comment (meisterT) [inline script/build_hosts:21]: We will be 7 DJ people in Dhaka - perhaps we can at least make room in the hosts file for this even if we don't end up with 7 machines?

File: script/build_hosts [hunk 4]
```diff
@@ -80,9 +84,9 @@ judges:
         print_server("balloonmanager", "10.4.4.149", others)
         # print_server("kattis", "10.3.3.212", others)
         # print_server("kattisbackup", "10.3.3.213", others)
-        # print_server("pc2-shadowccs", "10.3.3.214", others)
+        print_server("pc2-shadowccs", "10.3.3.214", others)
         # print_server("domjudge-shadowccs", "10.3.3.215", others)
-        # print_server("domserver", "10.3.3.216", others)
+        print_server("domserver", "10.3.3.216", others)
```
Comment (meisterT) [inline script/build_hosts:89]: do we need a `domserver-backup` or similar?

File: script/build_hosts (hunk 1 above)
Comment (troy2914) [reply script/build_hosts:22]: pc2 this year.

File: script/build_hosts (hunk 2 above)
Comment (troy2914) [reply script/build_hosts:20]: done in next push.

File: script/build_hosts (hunk 4 above)
Comment (troy2914) [reply script/build_hosts:89]: will do

File: script/build_hosts (hunk 3 above)
Comment (troy2914) [reply script/build_hosts]: sure

File: lastminute.yml [hunk 5]
```diff
@@ -8,7 +8,9 @@
       when: ldap_domain is defined
     - role: hyperthreadingdisabled
     - role: reverseproxy
+      when: enable_reverseproxy is defined
```
Comment (vmcj) [inline lastminute.yml:11]:
```suggestion
      when: enable_reverseproxy is defined and enable_reverseproxy
```

**Review by meisterT — APPROVED** [review]

File: lastminute.yml (hunk 5 above)
Comment (troy2914) [reply lastminute.yml]: updated

**Pull request merged**